packages/web-components/@sfx/search-box/CHANGELOG.md

```diff
@@ -6,5 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- SFX-148: Added interaction tests for the SearchBox component.
```

Comment:
I would put this below SFX-151. Adding the interaction tests before the component it's testing doesn't make sense.